client/src/components/subscription/SelectPlan.jsx

```diff
@@ -49,8 +37,7 @@ const SelectPlan = ({ setMealPlan }) => {
   };
 
   return (
-    <ThemeProvider theme={myTheme}>
-      <CssBaseline />
```

Comment:
@afong888 I realized ThemeProvider and CssBaseline weren't needed here because I'm already wrapping all my components with them from Subscribe.jsx